onnxruntime/core/optimizer/layer_norm_fusion.cc

```diff
@@ -139,6 +144,14 @@ data are casted to float/double to calculate for precision, so if there is any C
 Such Cast Op can be the input of the sub-graph, or an Cast Op between the Div and Mul nodes.
 */
 Status LayerNormFusion::ApplyImpl(Graph& graph, bool& modified, int graph_level, const logging::Logger& logger) const {
+  const auto& version_map = graph.DomainToVersionMap();
+  const auto& onnx_version = version_map.find(kOnnxDomain);
+  bool layernorm_fusion_flag = (onnx_version != version_map.end() && onnx_version->second >= 16);
+  const auto compatible_providers = GetCompatibleExecutionProviders();
+  if ((optimize_level == 1 && !layernorm_fusion_flag) || (optimize_level == 2 && layernorm_fusion_flag)) {
```

Comment:
What's the purpose of the optimize_level == 2 check?

Reply:
"layernorm_fusion_flag==True" means that the opset requirement is satisfied, and the layernorm_fusion optimization will be done in Level1. So there is no need to execute layernorm_fusion in Level2?